src/main/java/reposense/authorship/FileInfoAnalyzer.java

```diff
@@ -78,9 +79,10 @@ private static FileResult generateFileResult(FileInfo fileInfo) {
     }
 
     /**
-     * Sets the {@code Author} for each line in {@code fileInfo} based on the git blame analysis on the file.
+     * Sets the {@code Author} and {@code Date} for each line in {@code fileInfo} based on the git blame analysis
+     * on the file.
      */
-    private static void aggregateBlameAuthorInfo(RepoConfiguration config, FileInfo fileInfo) {
+    private static void aggregateBlameAuthorModifiedAndDateInfo(RepoConfiguration config, FileInfo fileInfo) {
```

Comment:
Not sure if this change is required. Since it's technically a part of the `aggregateBlame`. @fzdy1914 your thoughts?

Reply:
how about `aggregateBlameInfo`?